certbot-nginx/certbot_nginx/obj.py

```diff
@@ -110,7 +110,7 @@ class VirtualHost(object):  # pylint: disable=too-few-public-methods
 
     """
 
-    def __init__(self, filep, addrs, ssl, enabled, names, raw):
+    def __init__(self, filep, addrs, ssl, enabled, names, raw, path):
```

Comment:
Yup, that's what I intended.

Barring pathological cases of server blocks inside server blocks? I don't think that's possible though, and even then, it will only choose the top-most one, which I think we can probably be fine with.